.github/workflows/main.yaml

```diff
@@ -23,6 +26,13 @@ jobs:
           path: "/home/runner/repo-cache/"
           key: repo-cache
 
+      - name: Mount Go cache
+        uses: actions/cache@v4
+        with:
+          path: "/home/runner/go-mod-cache/"
+          key: go-mod-cache-${{ runner.os }}-${{ hashFiles('go.mod', 'go.sum', 'deps.bzl') }}
```

Comment:
We should also include MODULE.bazel and WORKSPACE, which can result in `fetch-repo` and Gazelle changing. I'm not completely convinced that's everything we need in here, but we can give it a try.

Reply:
added